reverse_no_py3/revno.py

```diff
@@ -0,0 +1,5 @@
+n=input("Enter a number:")
```

Comment:
move file to existing reverse a no directory.

Reply:
done